src/corelibs/U2Formats/src/GenbankPlainTextFormat.cpp

```diff
@@ -85,19 +85,32 @@ bool GenbankPlainTextFormat::isStreamingSupport() {
 }
 
 bool GenbankPlainTextFormat::readIdLine(ParserState *st) {
-    if (!st->hasKey("LOCUS")) {
+    if (!st->hasKey(DNAInfo::LOCUS.toLocal8Bit())) {
         QByteArray rawData(st->buff);
-        int locusStartPos = rawData.indexOf("\nLOCUS");
-        if (-1 == locusStartPos) {
-            st->si.setError(tr("LOCUS is not the first line"));
-            return false;
-        } else {
+        int locusStartPos = rawData.indexOf(QString("\n%1").arg(DNAInfo::LOCUS));
```

Comment:
rawData is QByteArray and parameter of the indexOf is QString. The conversion is implicit.
I would suggest to cache LOCUS.toLocal8Bit from line 88 and re-use that QByteArray here.

Reply:
Done